libs/muddle/src/router.cpp

```diff
@@ -831,6 +831,12 @@ void Router::RoutePacket(PacketPtr const &packet, bool external)
       return;
     }
 
+    if (kademlia_routing_)
```

Comment:
Ed's kademlia routing changes - ask him

Reply:
This is a duplicate of the line above and they are both wrong. I will fix this in a subsequent PR (along with additional updates)